.changelog/_14.txt

```diff
@@ -0,0 +1,3 @@
+```release-note: bug
+core: Linux packages now have vendor label and set the default label to HashiCorp. This fix is implemented for any future releases, but will not be updated for historical releases.
```

Comment:
we typically use `core` to mean core Nomad functionality, so changing this would help differentiate.
```suggestion
build: Linux packages now have vendor label and set the default label to HashiCorp. This fix is implemented for any future releases, but will not be updated for historical releases.
```